src/pages/cli/project/troubleshooting.mdx

```diff
@@ -0,0 +1,180 @@
+  * GraphQL schema Client-Side Code generation: The Amplify CLI GraphQL workflow allows developers to generate client side code for web and mobile clients using the amplify codegen command. 
+
+## Further Reading
+The [AWS Knowledge Center](https://aws.amazon.com/premiumsupport/knowledge-center/) provides a deep dive and prescriptive guidance for building and managing scalable. For debugging Amplify deployed services, the sections for AppSync, API Gateway, DynamoDB are highly recommended. 
+
+## Troubleshooting Amplify CLI Push Failures
+<Callout information>
+Error Messages: “An error occurred during the push operation”
+</Callout>
+
+The [`amplify push`](https://docs.amplify.aws/cli/start/workflows/#amplify-push) command performs the following steps before completing successfully such as the following in the case of a project using a GraphQL API:
+
+1. Generate CloudFormation for deployment of resources to AWS.
+2. Generate VTL & associated CloudFormation for the GraphQL API by running amplify api gql-compile. 
+3. Build packages and upload the deployment artifacts into the application's deployment bucket. 
+4. It then uses the AWS CloudFormation SDK to deploy the CloudFormation stack into the deployment account.
+5. On a successful deployment of the resources to AWS, the Amplify CLI copies the deployment package in the name #current-cloud-backend.zip. This file is the source of truth for your deployment. Hence even if your local folder gets deleted or corrupted, you can always go back to your “last successful deployment” using the [`amplify pull`](https://docs.amplify.aws/cli/start/workflows/#amplify-pull) command.
+###### fig.1.0 Amplify Folders created in the Deployment Bucket
+![ts-guide-amplify-deployment-folders](/images/ts-guide-amplify-deployment-folders.png)
+
+The following section shows some examples on how to debug `amplify push` failures: 
+
+### Handling Push Failures from Manual Changes to Cloud Resources - Drift 
+<Callout warning>
+Warning** 
+Once an application is deployed to the cloud using Amplify CLI the cloud resources like DynamoDB tables, Cognito Roles, IAM policies should never be updated manually from the AWS console of the services. 
+</Callout>
+
+The Amplify CLI cannot track manual changes applied to the resources through the AWS console and hence subsequent deployment to the cloud may fail. This state of the system where the actual resource state diverges from the CloudFormation stack is known as “CloudFormation Drift”. Currently there is no automated process to revert the system back to the earlier state and hence each resource has to be inspected through the AWS console for “Drift” in its CloudFormation stack and the “Drift” has to be reversed manually.
```

Comment:
Same comment here about singling out the AWS console.

Reply:
I think the last two uses of `"Drift"` can be rewritten as drift (no quotes, lowercase).